pkg/monitortests/kubeapiserver/legacykubeapiservermonitortests/early_e2e_disruption.go

```diff
@@ -0,0 +1,65 @@
+package legacykubeapiservermonitortests
+
+import (
+	"fmt"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"regexp"
+	"time"
+
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+)
+
+// amount of time after E2E tests start that we consider "early"
+const earlyE2ECheckDuration = 3 * time.Minute
+
+func testEarlyE2EAPIServerDisruption(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
+	const testName = "[Jira:\"kube-apiserver\"] API servers should not experience disruption near the start of E2E testing"
+
+	// regex to match the backends we care about
+	var earlyE2EBackendRe = regexp.MustCompile(`^(cache-)?(openshift|kube|oauth)-api-`)
+
+	var periodStart, periodEnd time.Time // the time period we're interested in disruption
+	for _, event := range events {
+		if event.Source == monitorapi.SourceE2ETest {
+			periodStart = event.From
+			periodEnd = event.From.Add(earlyE2ECheckDuration)
+			break
+		}
+	}
+
+	junits := []*junitapi.JUnitTestCase{}
+	if periodStart.IsZero() {
+		return junits // no e2e tests ran, no point in this test
```

Comment:
would it be common practice to have a junit with a skip message here instead of nothing?

Reply:
Historically we'd just return a pass, but Abu just committed what afaik is the first monitortest using a skip last week or so. https://github.com/openshift/origin/pull/29034/files